src/include/duckdb/transaction/transaction_context.hpp

```diff
@@ -48,6 +48,7 @@ class TransactionContext {
 	}
 
 	idx_t GetActiveQuery();
+	idx_t GetIncrementalIndex();
```

Comment:
Can we move this to the `ObjectCache` instead? you could e.g. do this per query number, and then reset when a new query number is encountered